packages/concerto-core/lib/introspect/stringvalidator.js

```diff
@@ -34,7 +35,7 @@ class StringValidator extends Validator{
 
     /**
      * Create a StringValidator.
-     * @param {Field} field - the field this validator is attached to
+     * @param {Field | ScalarDeclaration} field - the field or scalar declaration this validator is attached to
```

Comment:
We appear to be inconsistent in our JSDoc types, in some places we use alternation (`Field | ScalarDeclaration`), in others we use a common supertype (`Object`).

Reply:
Addressed ✅